.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: 'Stale issues and PRs handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6.0.1
+        id: stale
+        with:
+          stale-issue-message: 'This issue has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
+          stale-pr-message: 'This PR has gone 30 days without any activity and meets the project’s definition of "stale". This will be auto-closed if there is no new activity over the next 30 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!'
+          stale-issue-label: 'stale'
+          days-before-stale: 30
+          days-before-close: 30
+          exempt-pr-labels: 'not_stale'
+          exempt-issue-labels: 'not_stale'
+      - name: Print outputs
+        run: echo ${{ join(steps.stale.outputs.*, ',') }}
```

Comment:
what's this?

Reply:
my bad, I didn't want to include it. I got it from their repo [stale.yml#L18](https://github.com/actions/stale/blob/0d6f83007122d952cf222d03c8a3f6460c3a80fe/.github/workflows/stale.yml#L18). It prints some debug lines https://github.com/actions/stale/actions/runs/3238841484/jobs/5307512618

Reply:
removed